core/drand_daemon_test.go

```diff
@@ -7,10 +7,11 @@ import (
 	"testing"
 	"time"
 
+	"github.com/stretchr/testify/assert"
+
 	"github.com/drand/drand/common/scheme"
 	"github.com/drand/drand/log"
 	"github.com/drand/drand/test"
-	"github.com/stretchr/testify/assert"
 
 	"github.com/stretchr/testify/require"
```

Comment:
Does this really pass our linter? I'd expect only 3 groups, not 4.

Reply:
Good catch, fixed.